test/keepalive_test.dart

```diff
@@ -49,7 +49,7 @@ void main() {
       services: [FakeEchoService()],
       keepAliveOptions: serverOptions,
     );
-    await server.serve(address: 'localhost', port: 8081);
+    await server.serve(address: 'localhost', port: 8082);
```

Comment:
What's w/ this change? Necessary?

Reply:
This fixes the flaky keepalive tests.

Reply:
I think 0 is fine, but you need some consistency along tests, I can pass 0 in the failing one, but you should consider another PR to fix all the random port passed around (unless they are really needed) 
<img width="794" alt="image" src="https://github.com/user-attachments/assets/32732543-5995-4954-ba90-976d6003da4a">

Testing with 0, if the tests passes I will push the change for keepalive_test

Reply:
@mraleph pushed `keepalive_test.dart` with port 0

Reply:
Thanks! We should definitely change all the tests which don't need a fixed port to use `0` as a port. This kinda slipped under the radar in reviews. We'll do it in another PR.